golang/postgresql.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/georgysavva/scany/v2/pgxscan"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/segmentio/ksuid"
+	"net/http"
+	"time"
+)
+
+type PostrgeSQL struct {
```

Comment:
There is a typo here, and at all places where this gets referenced.

```suggestion
type PostgreSQL struct {
```

Reply:
applied